### ISSUE TITLE
Feature Request: browserlist lockfile

### ISSUE THREAD

**Issue by xiaoxiangmoe** [browserslist/browserslist#316]
Target browsers changes automatically over time. We need lockfile to keep building compatibility and control update.

**Comment by ai** [comment]
You can use `yarn.lock` or `package-lock.json`. The same `browserslist` and `caniuse-lite` will always return the same result (expect `last 2 years` query).

Will this solution work for you?

**Comment by JamiesonRoberts** [comment]
I would question the wisdom of this. Yes target browser change over time, but so do browser usage stats. Most browsers auto update these days, so unless you specifically need to support specific browser sets, there isn't much use for this. Also, with the rules you can explicitly set minimum version requirements. 

For example. `ie >= 8` would include IE8 through IE11. Yes the documentation speaks to restricting older versions OUT of the list, but it allows you to do both types of equality checks. Set your baselines and then do greater than checks and it will have the same effect as having a lock file, but without adding another file to your repo.

**Comment by ai** [comment]
@JamiesonRoberts do you lock `package-lock.json` or `yarn.lock`?

Browserslist doesn’t download browsers stats. It uses browsers stats from `caniuse-lite` npm package. If you lock `caniuse-lite` version by `package-lock.json` or `yarn.lock` you will have the same browsers every time.

**Comment by JamiesonRoberts** [comment]
@ai Yep I do. I was more trying to provide a constructive option for the original person asking about keeping fixed support levels regardless of the stat set being used.
